WeakAurasOptions/OptionsFrames/MoverSizer.lua

```diff
@@ -450,6 +464,32 @@ local function ConstructMoverSizer(parent)
       region:StartMoving()
       mover.isMoving = true
       mover.text:Show()
+      -- build list of alignment coordinates
+      mover.align = {
+        x = {},
+        y = {}
+      }
+      local x, y = {}, {}
+      for k, v in pairs(WeakAuras.displayButtons) do
+        if k ~= data.id and v.view.visibility ~= 0 then
+          tinsert(x, v.view.region:GetLeft())
+          tinsert(x, v.view.region:GetRight())
+          tinsert(y, v.view.region:GetTop())
+          tinsert(y, v.view.region:GetBottom())
+        end
+      end
+      table.sort(x)
+      table.sort(y)
+      for k, v in ipairs(x) do
+        if v ~= x[k+1] then
```

Comment:
So here, we probably want to do something like, if abs(v - x[k+1]) > 2 then

Basically throwing away anything that is almost the same as the next coordinate.

Reply:
On thinking again, nah what you have is actually fine.